platforms/hyperledger-fabric/configuration/roles/create/ca-tools/orderer/tasks/main.yaml

```diff
@@ -68,6 +68,16 @@
   tags:
     - notest
 
+#####################################################################################################################
+# This task Check if crypto materials exists in vault.
+- name: "Check if crypto materials exists in vault."
+  include_role:
+    name: check/crypto_materials
+  vars: 
+    namespace: "{{ item.name | lower }}-net"
```

Comment:
What is the point of this check from Ansible? It is not telling the user what is missing and will succeed anyway. The Issue should only check in the crypto creation job, and fail the job if the secrets were not saved.

Reply:
This role contains waits for various certs like orderertls, msp etc
So, what I have asked @mgCepeda to do is, to have this task in `shared/configuration/roles/check/setup`
![image](https://user-images.githubusercontent.com/56861721/150502572-3e89b72e-5680-40f4-a3a2-cb5ebad4b64e.png)

And then flatten out the contents of `check/crypto_materials` into this same file `platforms/hyperledger-fabrc/configuration/roles/ca-tools/orderer/tasks/main.yaml` as these
![image](https://user-images.githubusercontent.com/56861721/150503922-eb05c52b-7664-41b5-916c-dee475848422.png)

Reply:
So has the wait for crypto jobs completion been removed?

Reply:
We need the waits, because the crypto creation happens via one of the containers of `ca-tools`